fix(gen): add opentofu as terraform alternative (#4421)
scaleway/docs-content: tutorials/use-cockpit-with-terraform/index.mdx

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- [Configured an SSH key](/organizations-and-projects/how-to/create-ssh-key/#how-to-generate-an-ed25519-ssh-key-pair-on-macos-and-linux)

> - [Installed Terraform/OpenTofu](https://www.terraform.io/downloads.html) on your local machine

Author: ```suggestion
- [Installed Terraform](https://www.terraform.io/downloads.html) or [OpenTofu](https://opentofu.org/docs/intro/install/) on your local machine
```

    ```
    cd cockpit-terraform
    ```
2. Run the following command to download the dependencies and initialize Terraform/OpenTofu:

    ```
     terraform init
    ```
3. Run the following command to deploy your infrastructure. Terraform/OpenTofu generates an execution plan showing you what actions are about to be performed.

    ```
     terraform apply
    ```

    ```

>     Terraform/OpenTofu used the selected providers to generate the following execution plan.

Reviewer: ```suggestion
    Terraformused the selected providers to generate the following execution plan.
```

    Resource actions are indicated with the following symbols:
      + create

>     Terraform/OpenTofu will perform the following actions:

Reviewer: ```suggestion
    Terraform will perform the following actions:
```
